oe_contact_forms.module

```diff
@@ -198,6 +198,20 @@ function oe_contact_forms_form_contact_form_form_alter(&$form, FormStateInterfac
   }
 
   $form['#entity_builders'][] = '_oe_contact_forms_contact_form_builder';
+  $form['#submit'][] = '_oe_contact_forms_submit_handler';
+}
+
+/**
+ * Custom submit handler to prevent redirect.
+ *
+ * @param array $form
+ *   The form.
+ * @param Drupal\Core\Form\FormStateInterface $form_state
+ *   The form state.
+ */
+function _oe_contact_forms_submit_handler(array &$form, FormStateInterface $form_state): void {
+  // It prevents any redirection.
+  $form_state['redirect'] = FALSE;
```

Comment:
could you rather do this change here https://github.com/openeuropa/oe_contact_forms/blob/OPENEUROPA-3239/src/Form/ContactMessageForm.php#L67 as described in the ticket,

and before altering make sure getRedirectPath is empty

Reply:
Okay I moved the code.